sys/net/gnrc/routing/rpl/gnrc_rpl.c

```diff
@@ -319,7 +348,9 @@ void _dao_handle_send(gnrc_rpl_dodag_t *dodag)
     if ((dodag->dao_ack_received == false) && (dodag->dao_counter < GNRC_RPL_DAO_SEND_RETRIES)) {
         dodag->dao_counter++;
         gnrc_rpl_send_DAO(dodag->instance, NULL, dodag->default_lifetime);
-        dodag->dao_time = GNRC_RPL_DEFAULT_WAIT_FOR_DAO_ACK;
+        evtimer_del(&gnrc_rpl_evtimer, (evtimer_event_t *)&dodag->dao_event);
+        ((evtimer_event_t *)&(dodag->dao_event))->offset = GNRC_RPL_DEFAULT_WAIT_FOR_DAO_ACK * MS_PER_SEC;
```

Comment:
mhm, for the DAO delays you specify them as MS already, here `MS_PER_SEC` is used ... IMHO would be better to have the consistent.